portfolio/src/main/webapp/script.js

```diff
@@ -128,8 +128,10 @@ function showSlides(n) {
  * Fetch data from the server.
  */
 function getServerData() {
-  fetch('/data').then(response => response.text()).then((myJSON) => {
-    document.getElementById('data-container').innerText = myJSON;
-  });
+  fetch('/data')
+    .then(response => response.text())
+      .then( (myJSON) => document
```

Comment:
I think the brackets you had here were good, the dots in this one were all chaining, not jumping up and down levels, so it's fine to all be on the same line.

Reply:
ok got it!